proto/proto/transaction.proto

```diff
@@ -22,6 +22,11 @@ message Transaction {
   crypto.MerkleRoot anchor = 3;
 }
 
+// A clue for use with Fuzzy Message Detection.
+message Clue {
+  bytes inner = 1;
+}
+
```

Comment:
All the proto organization is a mess (let's clean it up after we get feature complete) but this probably belongs in `crypto.proto`, similar to the signature protos.